cloudconfig/cloudinit/cloudinit_centos.go

```diff
@@ -183,11 +183,12 @@ func (cfg *centOSCloudConfig) getCommandsForAddingPackages() ([]string, error) {
 		cmds = append(cmds, addPackageMirrorCmd(cfg, newMirror))
 	}
 
+	pkgCmder := cfg.paccmder[jujupackaging.YumPackageManager]
```

Comment:
This guaranteed to always exist?

Reply:
Yes, we set it up here: https://github.com/juju/juju/pull/10602/files/321e9305cdfba10805492f9e7e5aa5077f2c2663#diff-69cc37668e4f4d65f4b1dea9a62219abR439